packages/frontend/src/fetcher/on-chain/index.ts

```diff
@@ -171,6 +161,97 @@ class Fetcher implements IPartialFetcher {
         }, {});
     }
 
+    public async isAnswerer({
+        publicClient,
+        realityV3Address,
+        questionId,
+        address,
+    }: IsAnswererParams): Promise<boolean> {
+        if (!realityV3Address || !questionId || !address) return false;
+        await this.validate({ publicClient });
+
+        const realityContract = getContract({
+            abi: REALITY_ETH_V3_ABI,
+            address: realityV3Address,
+            publicClient,
+        });
+        const reopener = await realityContract.read.reopener_questions([
+            questionId,
+        ]);
+
+        let originalQuestionId = questionId;
+        if (reopener) {
+            originalQuestionId = await this.fetchOriginalQuestionId(
+                publicClient,
+                realityV3Address,
+                questionId
+            );
+        }
+
+        const questionsHistory = await this.fetchQuestionsHistory(
+            publicClient,
+            realityV3Address,
+            originalQuestionId
+        );
+
+        let answerer = false;
+        let toBlock = await publicClient.getBlockNumber();
+        let fromBlock = toBlock - LOGS_BLOCKS_SIZE;
+        while (true) {
+            const newAnwersEventLogs = await publicClient.getLogs({
+                address: realityV3Address,
+                event: parseAbiItem(
+                    "event LogNewAnswer(bytes32 answer,bytes32 indexed question_id,bytes32 history_hash,address indexed user,uint256 bond,uint256 ts,bool is_commitment)"
+                ),
+                args: {
+                    question_id: questionsHistory.map(
+                        (question) => question.questionId
+                    ),
+                    user: address,
```

Comment:
Very important, subgraphs store addresses lowercase. Address needs to be lowercased.